### ISSUE TITLE
Atelier "Ramène ta #data"

### ISSUE THREAD

**Issue by garaud** [pybdx/meetup-hub#8]
# Atelier "Ramène ta #data"

* Damien G. organisateur des meetups Python Bordeaux
* Organisateur / facilitateur
* Le type de format : atelier en plusieurs sessions
* Si vous souhaitez de l'aide et préparer un sujet à plusieurs mains : à plusieurs, c'est mieux !

En option, vous pouvez ajouter :

* Le type d'audience : à partir de débutant·e avec expérience en programmation
* Disponibilité(s) TODO

## Description

Je propose de monter un atelier dont l'objectif est de manipuler de la donnée
(de tout type) en **Python**.

La donnée est partout : industrie, e-commerce, capteurs, sur votre mobile, etc. Avant de
pouvoir extraire une information utile et pertinente, il est nécessaire de la traiter
de manière efficace. Dans la mouvance des *hastags hype* comme *deep learning*, *machine
learning*, *data mining* et autres choses en *ning*, la donnée doit pouvoir être lue et
traitée par ces algorithmes. C'est un travail fastidieux et néamoins intéressant !

Je vous propose de manipuler de la donnée (open data?) avec des boites à outil et
autres bibliothèques **Python** populaires telles que [pandas](https://pandas.pydata.org/), [matplotlib](https://matplotlib.org/), [jupyter](https://jupyter.org/), [etc](https://github.com/pydata).

L'idée est de travailler **ensemble** sur un ou plusieurs sujets sélectionnés et de
mettre les mains dans du code, produire de la donnée et des indicateurs à
visualiser...

Tu as une idée ? Ramène ta data !

Mots-clés :

- bienveillance
- nettoyage
- exploration
- visualisation
- préparation pour Machine Learning

**À faire** :

- déterminer le nombre max de personnes (~15, 20 ?)
- proposer un sujet avec un jeu open data manipulable aisément
- début de code pour manipuler la data choisie

**Comment by garaud** [comment]
cc @armgilles ça fait longtemps que ça traine dans ma tête. Qu'en penses-tu ?

Comme la dernière partie de la descirption "à faire" le suggère, faudrait :

- trouver un jeu de donnée (abordable par tous). En open data c'est mieux
- se donner un objectif

Quelques idées

## Datatourisme

Manipulation d'un bon gros JSON avec des lieux touristiques géolocalisées. À part faire quelques stats sur les différents types de lieux et un peu de carto, j'ai pas trop d'idées. Et le schéma du modèle de données (web sémantique, ontologie) a l'air bien velu :slightly_smiling_face: 

* https://www.data.gouv.fr/fr/datasets/donnees-touristiques-de-la-base-datatourisme/
* un peu de code python ici https://github.com/cquest/datatourisme

## Airbnb & qualité air à Bordeaux

J'ai trouvé des séries temporelles de capteurs de la qualité de l'air (géolocalisés) et aussi une liste de airbnb. Ça m'a donné une idée : "zone des logements Airbnb où la qualité de l'air est la meilleure". Ça n'a pas un très grand intérêt en soi mais je trouve ça fun !

* qualité air Bordeaux Métropole https://datacat.datalocale.fr/distribution/832087 j'ai aussi Mérignac, Talence, Bastide.
* Airbnb https://datacat.datalocale.fr/distribution/839285 ~50,000 lignes qui datent de 2017.

**Comment by remic33** [comment]
Hello,
J'ai pas mal de jeux de données ici des 'projects' de Datacamp, ils sont généralement utilisé pour apprendre à nettoyer et traiter des données. 
Exemple de projets réutilisable : https://github.com/Oysiyl/DataCamp_Solutions_Python/blob/master/A%20New%20Era%20of%20Data%20Analysis%20in%20Baseball/notebook.ipynb

**Comment by garaud** [comment]
@remic33 super !! Merci beaucoup pour ton lien. Une ressource intéressante en effet. J'ai parfois un peu de mal à trouver des jeux de données & des problématiques "fun" pour s'exercer. Ça a l'air d'être des bons exemples :slightly_smiling_face:

**Comment by remic33** [comment]
Yes j'en ai une vingtaine, ça peut aussi bien être du nettoyage que du traitement d'images et de la classifications. Je vais mettre sur github tous mes projects datacamp dans le week-end.

Pas de deep learning par contre... je suis en train de faire le programme fast.ai, quand j'aurais terminé je ferais quelques retours à ce sujet.

**Comment by garaud** [comment]
Merci pour ces infos et ces futurs retours.

J'ai jeté un oeil aux solutions DataCamp. C'est bien, il y a toujours les données associées au notebook. Par contre, il n'y a pas trop d'info sur la donnée (contenu, objectifs, etc.) à moins de lire le notebook et de voir assez vite ce qu'il en fait. Et dans ce cas, on perd un peu l'aspect exploratoire.
